app/controllers/agent/ConfirmClientUTRController.scala

```diff
@@ -39,7 +39,8 @@ import scala.concurrent.{ExecutionContext, Future}
 class ConfirmClientUTRController @Inject()(confirmClient: confirmClient,
                                            val authActions: AuthActions,
                                            val auditingService: AuditingService,
-                                           val sessionDataService: SessionDataService)
+                                           val sessionDataService: SessionDataService,
+                                           val ITSAStatusService: ITSAStatusService)
```

Comment:
if we can revert this to original version as we don't need ITSAStatusService within this controller as its not in use?